docs/dev/developer-guides/transactions/transactions.md

```diff
@@ -1,61 +1,71 @@
+>  - `maxFeePerGas`: the maximum fee per unit of gas willing to be paid for the transaction (inclusive of baseFeePerGas and maxPriorityFeePerGas).
 
-- **Receiver**: The recipient is the account's address to receive the transaction. The receiver can be a contract account or an externally owned account. Each transaction is aimed toward a specific recipient.
-- **Nonce**: This field displays the most recent transaction based on the account's counter, which maintains track of how many transactions it does. The network uses the transaction nonce to ensure that transactions are completed in the correct sequence.
-- **Gas price**: Most transactions necessitate the payment of a fee from the transaction's author. This cost is computed per unit of gas. The unit is Wei, a smaller ether unit.
-- **Gas limit**: The transaction author specifies the number of gas units used for the transaction. This is the total amount of gas that could be consumed.
-- **Value**: The quantity of Wei or Ether that the sender account wishes to transmit to the recipient is represented by the value.
-- **Data**: If the transaction receiver is a smart contract, the data contains information for the contract's functions to be executed. This comprises data with varying lengths.
-- **Signature**: A signature indicates who sent the communication. The signature is created when an externally owned account confirms and signs the transaction with its private key.
+## Nature of transactions
 
-### Transaction Types
+There are various different ways of transacting on the Ethereum blockchain:
 
-- Simple or asset transfers: This refers to the regular tokens transfer in the form of ether from one account to another.
+- Regular transactions: A simple transaction from one account to another such as transferring a token.
+- Contract deployment transactions: A transaction with no `to` address and the `data` field contains the contract code.  
+- Contract deployment on zkSync Era is different from Ethereum.
+  - Ethereum: Contract deployment occurs when a user sends a transaction to the zero address `(0x000...000)` where the `data` field contains the contract bytecode concatenated with constructor parameters.
+  - zkSync Era: To deploy a contract on zkSync, a user calls the `create` function of the [ContractDeployer](../system-contracts.md#contractdeployer) and provides the hash of the contract to be published, as well as the constructor arguments. The contract bytecode is added to the `factory_deps` field of [EIP-712 transaction types](#eip-712-0x71). If the contract is a factory (i.e. it can deploy other contracts), the bytecode of all child contracts should be included in `factory_deps`.
+- Find out more about [contract deployment](../../building-on-zksync/contracts/contract-development.md) on zkSync Era.
+- Contract execution: A transaction that interacts with an already deployed smart contract where the `to` address is the smart contract's address.
```

Comment:
How is this different from the "Regular transaction" above? Is the above only for a simple eth transfer? please clarify